### ISSUE TITLE
npm install fails when husky is not present in the workspace

### ISSUE THREAD

**Issue by rajiteh** [unional/path-equal#78]
seems that the postInstall hook requires husky to exist in the `node_modules/` however it is only defined as a devDependency, so it does not get pulled at install time.

```
npm ERR! code 127
npm ERR! path /path/to/project/node_modules/path-equal
npm ERR! command failed
npm ERR! command sh /tmp/postinstall-3826db94.sh
npm ERR! /tmp/postinstall-3826db94.sh: line 1: husky: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rajiteh/.npm/_logs/2022-10-28T02_21_29_799Z-debug-0.log
```

**Comment by unional** [comment]
Thanks! Will fix in in 5 mins.

**Comment by unional** [comment]
I wonder about that for quite a while. I thought it was wrong but no one complains about it. So I leave it as is. Thanks for reporting!

**Comment by unional** [comment]
It should be fixed in 1.2.4
